roles/cs.aws-rds/defaults/main.yml

```diff
@@ -1,14 +1,16 @@
 aws_rds_backup_retention: 5
 aws_rds_backup_window: 02:00-04:00
 aws_rds_maintenance_window: Mon:04:00-Mon:05:00
-aws_rds_auto_upgrade: true
+aws_rds_auto_minor_version_upgrade: yes
+aws_rds_allow_major_version_upgrade: no
 aws_rds_storage_size: 10
+aws_rds_storage_encrypt: no
+aws_rds_dedicated_encryption_key: no
 aws_rds_db_engine: MySQL
 aws_rds_availability_zone: "{{ aws_preferred_availability_zone }}"
 aws_rds_subnet_group_name: "{{ mageops_app_name }}"
 aws_rds_subnet_group_subnets: "{{ aws_vpc_subnet_ids }}"
-aws_rds_security_groups:
```

Comment:
Why this change?
It does not accept multiple groups anymore? If not - we need also to change the name of the variable.